tests/phpunit/tests/html-api/wpHtmlProcessorHtml5lib.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+/**
+ * Unit tests covering HTML API functionality.
+ *
+ * This test suite runs a set of tests on the HTML API using a third-party suite of test fixtures.
+ * A third-party test suite allows the HTML API's behavior to be compared against an external
+ * standard. Without a third-party, there is risk of oversight or misinterpretation of the standard
+ * being implemented in application code and in tests. html5lib-tests is used by other projects like
+ * browsers or other HTML parsers for the same purpose of validating behavior against an
+ * external reference.
+ *
+ * See the README file at DIR_TESTDATA / html5lib-tests for details on the third-party suite.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ *
+ * @since 6.5.0
+ *
+ * @group html-api
+ */
+class Tests_HtmlApi_Html5lib extends WP_UnitTestCase {
+	/**
+	 * The HTML Processor only accepts HTML in document <body>.
+	 * Do not run tests that look for anything in document <head>.
+	 */
+	const SKIP_HEAD_TESTS = true;
+
+	/**
+	 * Skip specific tests that may not be supported or have known issues.
+	 */
+	const SKIP_TESTS = array(
```

Comment:
I'm concerned about the amount of tests being skipped. 

IMO skipping a test is informative to denote where work is still needed or planned for the future (such as skipping until a dependency or bug fix lands).

Here `<head>` is not supported. Thus, these skips do not add value. Instead, they will consume time and resources, adding more messages to shift through and adding more time to the test suite (performance).

Instead of skipping over the `<head>` tests, is there a way to start the tests at the `<body>`, thus passing only the body's tests in the data provider's datasets?

Also commenting on the ticket to track this discussion (since GH code comments do not post to the ticket).